rust/windows-client/README.md

```diff
@@ -2,14 +2,28 @@
 
 This crate houses a Windows GUI client.
 
-## Building
+## Setup
 
-From this dir:
+1. Install the latest
+   [Visual Studio Community Edition](https://visualstudio.microsoft.com/downloads/)
+   and ensure **C++ support with Clang** is selected. Note: Visual Studio Code
+   won't have clang -- you need actual Visual Studio.
+1. Install rustup.
+1. Install the toolchain: `rustup show`
+1. Install clang for Windows.
```

Comment:
Oops, on the branch we just killed I listed how to install clang (llvm) with chocolatey: https://github.com/firezone/firezone/pull/2985/files#diff-d4ff8dd7a4701ab4796f6d89b6f9c623f092891fd71ef16f2820f1ac457b3376

Reply:
Yeah I'll just remove these steps needed for cross-compiling and compiling for aarch64.